src/css/components/_c-person.scss

```diff
@@ -18,53 +19,60 @@ $_person-photo: (
 	}
 }
 
+@mixin photo($size: tiny) {
```

Comment:
Could you update this to add `_person-` to the mixin, and then move it up to after `$_person-photo`? The system for organizing it this way is:

- All Sass logic applicable to only this partial declared first, followed by the partial's class declarations.
- Namespace Sass logic applicable to only this partial with `_` and `person-`
    - `_` indicates that's its a partial relevant to this partial only, and not "global" logic that lives in `[logic/](https://github.com/a11yproject/a11yproject.com/tree/main/src/css/logic)`.
    - `person-` indicates it's for the `c-person` component, which can be helpful for searching through the repo's code.

Reply:
I've made some tweaks here, thanks for the info about the preferred styles!